astropy/utils/iers/tests/test_iers.py

```diff
@@ -377,3 +377,52 @@ def test_iers_b_dl():
         assert 'UT1_UTC' in iersb_tab.colnames
     finally:
         iers.IERS_B.close()
+
+
+@pytest.mark.remote_data
+def test_iers_out_of_range_handling():
+    # Make sure we don't have IERS-A data available anywhere
+    clear_download_cache()
```

Comment:
This feels a little dangerous, as it affects the package's state globally. What if a user runs this test locally and has their cached all cleared by accident?

Reply:
It would indeed be better to use [astropy.config.set_temp_cache()](https://docs.astropy.org/en/stable/api/astropy.config.set_temp_cache.html#astropy.config.set_temp_cache).

Reply:
Perfect, will do.